cookbooks/travis_internal_base/recipes/default.rb

```diff
@@ -29,7 +29,7 @@
 include_recipe 'travis_sudo'
 
 package 'whois' do
-  action [:install, :upgrade]
+  action %i(install upgrade)
```

Comment:
i looked up this change for chef (ruby) syntax and still don't quite understand change.  All other changes LGTM :+1: